src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -292,6 +298,65 @@ public void mergeStagingLoadHistoryTable(Dataset dataset, String flightId)
     bigQueryProject.query(sqlTemplate.render());
   }
 
+  private static final String getLoadHistoryTemplate =
+      "SELECT * "
```

Comment:
Given that we're doing paging, I think it's safer to specify a sort here since SQL grammar doesn't guarantee sort order unless explicitly specified.  Could be as simple as sorting by fileId.

Also, it's weird that we don't track load time in the load history table...

Reply:
Reading the azure storage table docs, we probably want to sort on file id since storage tables don't allow for arbitrary sorting.  Records are returned order by partition key and then record key (which in our case is the file Id or something generated from the file Id I believe)